engines/registration_sharing/app/controllers/registration_sharing/rmt_to_rmt_controller.rb

```diff
@@ -0,0 +1,48 @@
+require_dependency 'registration_sharing/application_controller'
+
+module RegistrationSharing
+  class RmtToRmtController < ApplicationController
+
+    before_action :authenticate
+
+    def create
+      System.transaction do
+        system = RegistrationSharing::System.find_or_create_by(login: params[:login])
+
+        %w[password hostname registered_at created_at last_seen_at].each do |attribute|
```

Comment:
Why not use [strong parameters](https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters) here?

Reply:
Yes, good idea.